### PR TITLE
Refresh all GitSavvy view on branch checkout

### PR DESCRIPTION
**Pull request by gwenzek**
Proof of concept implementation of #835.

Add a `refresh_all_visible_views` method.
It iterates through all the current window groups and refresh the active view of the group.

### CODE REVIEW

File: core/commands/checkout.py [hunk 1]
```diff
@@ -37,7 +37,12 @@ def on_branch_selection(self, branch):
 
         self.git("checkout", branch)
         sublime.status_message("Checked out `{}` branch.".format(branch))
-        util.view.refresh_gitsavvy(self.window.active_view(), refresh_sidebar=True)
+        refresh_all_visible_views(self.window)
+
+def refresh_all_visible_views(window):
```
Comment (stoivo) [inline core/commands/checkout.py]: Can you move this into [common/util/view.py](https://github.com/divmain/GitSavvy/blob/master/common/util/view.py#L81)

File: core/commands/checkout.py [hunk 2]
```diff
@@ -37,7 +37,12 @@ def on_branch_selection(self, branch):
 
         self.git("checkout", branch)
         sublime.status_message("Checked out `{}` branch.".format(branch))
-        util.view.refresh_gitsavvy(self.window.active_view(), refresh_sidebar=True)
+        refresh_all_visible_views(self.window)
```
Comment (stoivo) [inline core/commands/checkout.py]: Move and rename to 
`util.view.refresh_all_gitsavvy`

**Comment by stoivo** [comment]
Also rebase from `dev`

File: core/commands/checkout.py (hunk 1 above)
Comment (gwenzek) [reply core/commands/checkout.py]: Done.

File: core/commands/checkout.py (hunk 2 above)
Comment (gwenzek) [reply core/commands/checkout.py]: Done.

**Comment by gwenzek** [comment]
As requested:
* rebased on dev
* moved the method to `common.util.view` and renamed it to `refresh_all_gitsavvy`
Improvements:
* only refresh the file list once (potentially costly)
* also use `refresh_all_gitsavvy` for `GsCheckoutNewBranchCommand` and `GsCheckoutRemoteBranchCommand`

**Comment by asfaltboy** [comment]
While I ❤️  the improvements that this change introduces, I feel this defaults to a very generic refresh of all views. For example, sometimes I want to open some GitSavvy (a graph dashboard, a diff) multiple times, on different branches, to compare the code/commits in the versions. I fear that refreshing all views may break this use case (though I haven't tested it yet).

Should we perhaps limit this only to the status or branch dashboards? Or do you see any other views where this use case fits?

**Comment by stoivo** [comment]
I think we can add the limit to only reload the interfaces. BranchInterface, RebaseInterface StatusInterface and TagsInterface. Any of those should always be as updated as possible.

**Comment by gwenzek** [comment]
I modified the method to only refresh interfaces.
Should I rename it to something like `refresh_gitsavvy_interfaces` ?

The build tools are failing during the installation of Package Control, seems unrelated to my changes:
`Fail to install Package Control.`

**Comment by gwenzek** [comment]
I rename the method to `refresh_gitsavvy_interfaces` since it only refresh interfaces now.
The build still fails at the Package Control step, so it's hard for me to tell if I have a messed up spacing or such.

**Pull request merged**